lib/actions/apiV2.js

```diff
@@ -682,7 +682,12 @@ export const findRoute = (params) =>
                 })
               })
 
-            filteredPatterns.forEach((pattern) => {
+            // Fallback for if the filtering removes all patterns
+            // If this happens, it is not possible to know which pattern to keep
+            ;(filteredPatterns.length > 0
```

Comment:
Is the semi colon intended?

Reply:
Unfortunately yes this is the one time the looney goose C non compliant semicolon free style we love these days shows its issues

Reply:
Did you mean *Loosey Goose* ?

Reply:
Could the semi colon go on the line previous?

Reply:
Prettier put it here